### DISCUSSION TITLE
Can I tell handlebars ignore content in a tag?

### DISCUSSION THREAD

**Discussion by namleasia** [handlebars-lang/handlebars.js#1794] [Q&A] [answered]
Example Handlebars template:
```html
<div>Hello {{doesWhat}}</div>
<template class="handlebars-template-for-client-side">
<div>Client {{doesWhat}}</div>
</template>
```
Can I tell Handlebars ignore content in `template` tag?
I want to use Handlebars on both SSR and CSR.

**Comment by jaylinski** [comment, marked as answer]
You have two options, as described here: https://handlebarsjs.com/guide/expressions.html#escaping-handlebars-expressions

An [example](https://handlebarsjs.com/playground.html#format=1&currentExample=%7B%22template%22%3A%22%3Cdiv%3EHello%20%7B%7BdoesWhat%7D%7D%3C%2Fdiv%3E%5Cn%3Ctemplate%20class%3D%5C%22handlebars-template-for-client-side%5C%22%3E%5Cn%3Cdiv%3EClient%20%5C%5C%7B%7BdoesWhat%7D%7D%3C%2Fdiv%3E%5Cn%3C%2Ftemplate%3E%5Cn%5Cn%3Cdiv%3EHello%20%7B%7BdoesWhat%7D%7D%3C%2Fdiv%3E%5Cn%7B%7B%7B%7Braw%7D%7D%7D%7D%5Cn%3Ctemplate%20class%3D%5C%22handlebars-template-for-client-side%5C%22%3E%5Cn%3Cdiv%3EClient%20%7B%7BdoesWhat%7D%7D%3C%2Fdiv%3E%5Cn%3C%2Ftemplate%3E%5Cn%7B%7B%7B%7B%2Fraw%7D%7D%7D%7D%22%2C%22partials%22%3A%5B%5D%2C%22input%22%3A%22%7B%5Cn%20%20doesWhat%3A%20%5C%22World%5C%22%2C%5Cn%7D%5Cn%22%2C%22output%22%3A%22%3Cdiv%3EHello%20World%3C%2Fdiv%3E%5Cn%3Ctemplate%20class%3D%5C%22handlebars-template-for-client-side%5C%22%3E%5Cn%3Cdiv%3EClient%20%7B%7BdoesWhat%7D%7D%3C%2Fdiv%3E%5Cn%3C%2Ftemplate%3E%5Cn%5Cn%3Cdiv%3EHello%20World%3C%2Fdiv%3E%5Cn%3Ctemplate%20class%3D%5C%22handlebars-template-for-client-side%5C%22%3E%5Cn%3Cdiv%3EClient%20%7B%7BdoesWhat%7D%7D%3C%2Fdiv%3E%5Cn%3C%2Ftemplate%3E%5Cn%22%2C%22preparationScript%22%3A%22Handlebars.registerHelper('raw'%2C%20function%20(options)%20%7B%5Cn%20%20%20%20return%20options.fn()%3B%5Cn%7D)%5Cn%22%2C%22handlebarsVersion%22%3A%224.7.7%22%7D).

**Reply by namleasia** [reply]
Thank you so much!
I had looked around this page but can't see it, my mistake.
